.github/workflows/tests.yml

```
@@ -147,7 +140,7 @@ jobs:
           popd
 
   test_pywavelets_linux_free_threaded:
-    name: linux-cp313t-with-scipy
+    name: linux-cp313t-free-threaded
```

```suggestion
    name: linux-cp313t-default
```

cp313t implies free-threaded Python already; this job runs the default `pytest` test suite.